<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_information</class>
 <widget class="QMainWindow" name="change_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image:url(back2.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-image:url(:/new/img/bgimg.png)</string>
   </property>
   <widget class="QGroupBox" name="password">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url()</string>
    </property>
    <property name="title">
     <string>修改密码</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>旧的密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="old_password"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>新的密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new_password"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>确认密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirm_password"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url()</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_change_password">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/*	qproperty-icon: url(./appIcons/location-1.png);
按钮普通态*/
QPushButton
{
	border:0.5px groove gray;
	border-radius:5px;
	padding:2px 4px;
	background-color:rgb(233 , 232 , 227);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:5px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:5px;
}</string>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
       <property name="icon">
        <iconset resource="img_need.qrc">
         <normaloff>:/new/img/钥匙.png</normaloff>:/new/img/钥匙.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_return">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/*	qproperty-icon: url(./appIcons/location-1.png);
按钮普通态*/
QPushButton
{
	border:0.5px groove gray;
	border-radius:5px;
	padding:2px 4px;
	background-color:rgb(233 , 232 , 227);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:5px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:5px;
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="icon">
        <iconset resource="img_need.qrc">
         <normaloff>:/new/img/返回.png</normaloff>:/new/img/返回.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>pushButton_return</zorder>
    <zorder>pushButton_change_password</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img_need.qrc"/>
 </resources>
 <connections/>
</ui>
